README.md

```diff
@@ -4,12 +4,71 @@
 
 # Identity Wallet
 
-```
-npm run storybook
-```
+This is a wallet built to hold OpenAttestation-compatible documents
+
+https://openattestation.com/docs/getting-started
+https://github.com/Open-Attestation/open-attestation
 
 ## Deployed Application
+
 ![Expo QR](https://api.qrserver.com/v1/create-qr-code/?size=250x250&data=exp://exp.host/@dlt/identitywallet)
 
 ## Deployed Storybook
+
 ![Expo QR](https://api.qrserver.com/v1/create-qr-code/?size=250x250&data=exp://exp.host/@dlt/identitywallet?release-channel=storybook-default)
+
+# Development
+
+### Install required dependencies
+
+```
+npm i
+```
+
+### To run storybook
+
+```
+npm run storybook
+```
+
+### To run application
+
+```
+npm start
+```
+
+Documents can be found at https://gallery.openattestation.com/
+
+### Test
+
+```
+npm run test
+```
+
+# Created Resources
```

Comment:
Could you explain me the goal of the different section ? For instance I dont understand why talking about caas renderer ?